CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 
 
+---
+## [7.0.0] - 2022-10-03
+### Added
+### Changed
+- Use CryptoCore v3.0
```

Comment:
I think we should talk a bit more of the implications of using CryptoCore, like backward incompatible keys that justify a new major version.

Reply:
I have added some more detail to the changelog